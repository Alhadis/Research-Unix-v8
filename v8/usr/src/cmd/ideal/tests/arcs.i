arcs
.nf
.IS
...radians
...libfile a
main{
a1:put arc{start = 6; midway = (5,-1); end = 4;};
a2:put arc{start = (1,-3); midang = 3.14159/2; end = (0,-4);im(center) = -3;};
a3:put arc{center = (4.5,-3); radius = 1.2; startang = 2; endang = 3;};
a4:put arc{center = (3,-6); start = (4,-5); endang = 0; midang = 3.14159;};
put b1: arc {center = 0; radius = 0.25; startang = 1; midang = 0.5; endang = 0;};
put b2: arc {center = 0; radius = 0.5; startang = 1; midang = 0; endang = 0.5;};
put b3: arc {center = 0; radius = 0.75; startang = 0.5; midang = 1; endang = 0;};
put b4: arc {center = 0; radius = 1; startang = 0; midang = 1; endang = 0.5;};
put b5: arc {center = 0; radius = 1.25; startang = 0.5; midang = 0; endang = 1;};
put b6: arc {center = 0; radius = 1.5; startang = 0; midang = 0.5; endang = 1;};
}
.IE
