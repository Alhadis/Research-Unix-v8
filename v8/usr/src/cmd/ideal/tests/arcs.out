arcs
.nf
.IS
...maxx 6
...maxy 1.26221
...minx -1.25
...miny -7.41421
...arc 0 0 1.5 0 0.810453 1.26221 0 1 1.5
...arc 0 0 0.675378 1.05184 1.09698 0.599282 1 6.78319 1.25
...arc 0 0 0.877583 0.479426 1 0 0.5 6.28319 1
...arc 0 0 0.658187 0.359569 0.75 0 0.5 6.28319 0.75
...arc 0 0 0.270151 0.420735 0.438791 0.239713 1 6.78319 0.5
...arc 0 0 0.25 0 0.135076 0.210368 0 1 0.25
...arc 3 -6 4 -5 4.41421 -6 0.785398 6.28319 1.41421
...arc 4.5 -3 4.00062 -1.90884 3.31201 -2.83066 2 3 1.2
...arc 0 -3 1 -3 1.19249e-08 -4 0 4.71239 1
...arc 5 0 4 -8.74228e-08 6 0 3.14159 6.28319 1
.IE
